app/src/main/java/com/github/swent/echo/viewmodels/event/EventViewModel.kt

```diff
@@ -35,6 +37,7 @@ constructor(
     val isEventNew = _isEventNew.asStateFlow()
     private val _organizerList = MutableStateFlow<List<String>>(listOf())
     val organizerList = _organizerList.asStateFlow()
+    val isOnline = networkService.isOnline
```

Comment:
I'm not sure it makes sense to get the `isOnline` parameter only once. Could it be a problem if the device goes offline after this state is read?

Reply:
The networkService returns a StateFlow, so the composable will be updated when the value changes